Content.Server/CartridgeLoader/CartridgeLoaderSystem.cs

```diff
@@ -79,17 +84,37 @@ public List<EntityUid> GetAvailablePrograms(EntityUid uid, CartridgeLoaderCompon
         if (!Resolve(uid, ref loader))
             return new List<EntityUid>();
 
-        //Don't count a cartridge that has already been installed as available to avoid confusion
-        if (loader.CartridgeSlot.HasItem && IsInstalled(Prototype(loader.CartridgeSlot.Item!.Value)?.ID, loader))
+        var unsecureAllowed = _configManager.GetCVar(CCVars.CrewManifestUnsecure);
```

Comment:
or this

Reply:
Reconsidered the approach. Now cartridge loader system doesn't know anything about special cases. Thank you.